backend/src/controllers/picker_options.js

```diff
@@ -0,0 +1,98 @@
+const log = require("../facilities/logging.js")(module.filename);
+const Model = require("../models/picker_options.js");
```

Comment:
Small thing but it seems like the app is using both camelCase and underscores as file naming conventions(with underscores being used for seeds and migrations). Maybe confer with the team about keeping it consistent(or having an agreed convention)

Reply:
I think we did have a conversation and most React things are camelcase and everything else is underscores.